src/views/layouts/main.handlebars

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+    </head>
+    <body>
+        {{{body}}}
```

Comment:
Handlebars use "triple-stash" to avoid HTML escaping - [handlebarsjs.com#html-escaping](http://handlebarsjs.com/#html-escaping).